lib/kitchen/transport/winrm.rb

```diff
@@ -122,9 +133,9 @@ def upload(locals, remote)
         def wait_until_ready
           delay = 3
           session(
-            :retries  => max_wait_until_ready / delay,
-            :delay    => delay,
-            :message  => "Waiting for WinRM service on #{endpoint}, " \
+            retries: max_wait_until_ready / delay,
```

Comment:
I actually don't think we'll be able to make this change - some people still use TK with Ruby 1.9